DEBUGGING.md

```diff
@@ -39,3 +41,23 @@ kubectl logs <YOUR_POD_NAME> | less -R
 ```
 
 where you would replace `<YOUR_POD_NAME>` with the name of the pod.
+
+## Debugging the vscode extension
+In order to debug the vscode extension, first [build the mirrord binary](TESTING.md#build-and-run-mirrord). Then run:
+```bash
+cd vscode-ext
+```
+```bash
+npm install
+npm run compile
+```
+Now you can just open the extension's code in vscode and run. Another vscode window will start. You can set breakpoints
+in the extension's code in the first window, and use the extension in the second window to reach the breakpoints.
+When in debug mode, the extension will automatically use the debug mirrord binary.
+
+If you want to see the layer's logs, [use the console](#mirrord-console) by setting the environment variable in the
```

Comment:
Is this the `MIRRORD_CONSOLE_ADDR=127.0.0.1:11233` env var? If so I would link to it

Reply:
That works